### PR TITLE
569 change containers gutter

### PR DESCRIPTION
**Pull request by ArekRado**

### CODE REVIEW

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss [hunk 1]
```diff
@@ -7,62 +7,62 @@
...
 
-    .firstCol {
-      @include media-breakpoint-down(sm) {
-        order: 2;
-        padding-top: 20px;
-      }
+  .firstCol {
+    @include media-breakpoint-down(sm) {
+      order: 2;
+      padding-top: 20px;
```
Comment (CodeMeNatalie) [inline src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: pt

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss [hunk 2]
```diff
@@ -7,62 +7,62 @@
...
-  .secondSection {
-    @include mt(7);
+.secondSection {
+  @include mt(7);
 
-    @include media-breakpoint-down(sm) {
-      margin-top: 90px;
-    }
+  @include media-breakpoint-down(sm) {
+    margin-top: 90px;
```
Comment (CodeMeNatalie) [inline src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: mt

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss [hunk 3]
```diff
@@ -7,62 +7,62 @@
...
+  }
 
-    .secondCol {
-      padding-left: 140px;
-      @include media-breakpoint-down(sm) {
-        margin-top: 20px;
-        padding-left: 0;
-      }
+  .secondCol {
+    padding-left: 140px;
```
Comment (CodeMeNatalie) [inline src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: pt

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss [hunk 4]
```diff
@@ -7,62 +7,62 @@
...
+    @include media-breakpoint-down(sm) {
+      margin-top: 20px;
```
Comment (CodeMeNatalie) [inline src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: mt

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss [hunk 5]
```diff
@@ -7,62 +7,62 @@
...
+      padding-left: 0;
```
Comment (CodeMeNatalie) [inline src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: pl

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss [hunk 6]
```diff
@@ -7,62 +7,62 @@
...
     }
   }
+}
 
-  .thirdSection {
-    @include mt(7);
-    margin-bottom: 90px;
+.thirdSection {
+  @include mt(7);
+  margin-bottom: 90px;
```
Comment (CodeMeNatalie) [inline src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: my?

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss [hunk 7]
```diff
@@ -7,62 +7,62 @@
...
 
-    @include media-breakpoint-down(sm) {
-      margin-bottom: 60px;
-      margin-top: 90px;
-    }
+  @include media-breakpoint-down(sm) {
+    margin-bottom: 60px;
```
Comment (CodeMeNatalie) [inline src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: my

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss [hunk 8]
```diff
@@ -7,62 +7,62 @@
...
 
-    .firstCol {
-      @include media-breakpoint-down(sm) {
-        order: 2;
-        padding-top: 20px;
-      }
+  .firstCol {
+    @include media-breakpoint-down(sm) {
+      order: 2;
+      padding-top: 20px;
```
Comment (CodeMeNatalie) [inline src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: pt

File: src/routes/Home/Home.module.scss [hunk 9]
```diff
@@ -4,17 +4,17 @@
...
+  height: 400px;
 
-    @include media-breakpoint-down(sm) {
-      height: 218px;
-      padding-top: 40px;
-      padding-bottom: 102px;
-    }
+  @include media-breakpoint-down(sm) {
+    height: 218px;
+    padding-top: 40px;
```
[the remark this answers is not in the document]
Comment (ArekRado) [reply src/routes/Home/Home.module.scss]: we don't have spacing 40px (only 30 and 50) ;(

File: src/routes/Home/Home.module.scss [hunk 10]
```diff
@@ -4,17 +4,17 @@
...
   width: 100%;
   overflow: hidden;
+}
 
-  .headingSection {
-    padding-top: 90px;
-    width: 100%;
-    height: 400px;
+.headingSection {
+  padding-top: 90px;
```
[the remark this answers is not in the document]
Comment (ArekRado) [reply src/routes/Home/Home.module.scss]: we don't have spacing 90px (only 75 and 120) ;(

File: src/routes/Home/components/Clients/Clients.module.scss [hunk 11]
```diff
@@ -10,60 +10,60 @@
     padding-top: 60px;
```
[the remark this answers is not in the document]
Comment (ArekRado) [reply src/routes/Home/components/Clients/Clients.module.scss]: we don't have spacing 60px (only 50 and 75) ;(

File: src/routes/Home/components/ClutchSection/ClutchSection.module.scss [hunk 12]
```diff
@@ -14,46 +14,46 @@
     padding-top: 60px;
```
[the remark this answers is not in the document]
Comment (ArekRado) [reply src/routes/Home/components/ClutchSection/ClutchSection.module.scss]: niet

File: src/routes/Home/components/ClutchSection/ClutchSection.module.scss [hunk 13]
```diff
@@ -14,46 +14,46 @@
...
+.cardsWrapper {
+  @include inline-flex(row, flex-start, space-between);
+  @include pb(2);
+  padding-top: 143px;
 
-    @include media-breakpoint-down(md) {
-      padding-top: 90px;
-    }
+  @include media-breakpoint-down(md) {
+    padding-top: 90px;
```
[the remark this answers is not in the document]
Comment (ArekRado) [reply src/routes/Home/components/ClutchSection/ClutchSection.module.scss]: niet

File: src/routes/Home/components/ClutchSection/ClutchSection.module.scss [hunk 14]
```diff
@@ -14,46 +14,46 @@
...
-      border-radius: 3px;
-      box-shadow: 0 10px 30px 0 rgba(0, 0, 0, 0.1);
-      padding: 40px;
+  .card {
+    @include mx(1);
+    width: 100%;
+    background-color: $color-white;
+    border-radius: 3px;
+    box-shadow: 0 10px 30px 0 rgba(0, 0, 0, 0.1);
+    padding: 40px;
```
[the remark this answers is not in the document]
Comment (ArekRado) [reply src/routes/Home/components/ClutchSection/ClutchSection.module.scss]: we don't have spacing 40px (only 30 and 50) ;(

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss (hunk 1 above)
Comment (ArekRado) [reply src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: we don't have spacing 20px (only 15 and 30) ;(

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss (hunk 2 above)
Comment (ArekRado) [reply src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: we don't have spacing 90px (only 75 and 120) ;(

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss (hunk 3 above)
Comment (ArekRado) [reply src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: we don't have spacing 140px (only 120 and 180) ;(

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss (hunk 4 above)
Comment (ArekRado) [reply src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: niet

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss (hunk 6 above)
Comment (ArekRado) [reply src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: niet

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss (hunk 7 above)
Comment (ArekRado) [reply src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: we don't have spacing 60px (only 50 and 75) ;(

File: src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss (hunk 8 above)
Comment (ArekRado) [reply src/routes/Home/components/WhatWeDo/WhatWeDo.module.scss]: we don't have spacing 20px (only 15 and 30) ;(

File: src/components/Header/Header.module.scss [hunk 15]
```diff
@@ -26,65 +26,65 @@
...
+    @include flex(col, flex-start, center);
+  }
+}
 
-    .nav {
-      @include flex(column);
-      position: relative;
+.nav {
+  @include flex(column);
+  position: relative;
```
Comment (CodeMeNatalie) [inline src/components/Header/Header.module.scss]: `@include relative`

File: src/components/Switch/Switch.module.scss [hunk 16]
```diff
@@ -10,25 +10,25 @@ $switch-height: 36px;
   padding: 4px;
   position: relative;
```
Comment (CodeMeNatalie) [inline src/components/Switch/Switch.module.scss]: `@include relative`

File: src/components/ToggleSwitch/ToggleSwitch.module.scss [hunk 17]
```diff
@@ -28,23 +28,23 @@ $toggle-width-sm: 85px;
...
-    @include responsive-prop(width, (
-      up md: $toggle-width-md * 0.6,
-      down sm: $toggle-width-sm * 0.6
-    ));
-    height: 100%;
-    border-radius: $border-radius;
-    background-color: $brand-primary;
+.sliderRound {
+  display: inline-block;
+  @include transition();
```
Comment (CodeMeNatalie) [inline src/components/ToggleSwitch/ToggleSwitch.module.scss]: `@include transition`

File: src/routes/Home/Home.module.scss [hunk 18]
```diff
@@ -4,17 +4,16 @@
   height: 100%;
   width: 100%;
```
Comment (CodeMeNatalie) [inline src/routes/Home/Home.module.scss]: mixin size

**Pull request merged**